WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -719,6 +719,7 @@ - (void)showBlogSelector
                                                                                        dismissHandler:dismissHandler];
     vc.title = NSLocalizedString(@"Select Site", @"");
     vc.displaysPrimaryBlogOnTop = YES;
+    vc.displaysCancelButton = [UIDevice isPhone];
```

Comment:
This doesn't account for multitasking:

![simulator screen shot 05 apr 2016 09 59 55](https://cloud.githubusercontent.com/assets/8739/14275036/2aaf9d86-fb15-11e5-8f7e-9e0c50e170c1.png)

Not a big deal here since tapping on a site will dismiss, but in general not a good idea to look at `UIDevice` anymore for UI decisions.